init.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Loads dbinit.sql into your running CockroachDB cluster
+cockroach sql --insecure --url="postgres://root@127.0.0.1:$MOVR_PORT" < dbinit.sql
+
+# Resets .env file, clearing out any variables that were previously set
+git co -- .env
```

Comment:
git: 'co' is not a git command. See 'git --help'.

Did you mean one of these?
        commit
        ca
        cm

Reply:
oops, that's an alias that I've set. It should be `git checkout -- .env`.

thanks for catching -- can update later today